src/cpp/session/CMakeLists.txt

```diff
@@ -725,26 +737,11 @@ if(NOT RSTUDIO_SESSION_WIN32 AND NOT RSESSION_ALTERNATE_BUILD)
    if(QUARTO_ENABLED)
       # install some quarto folders into Resources, as needed
       if(APPLE)
-         if (RSTUDIO_ELECTRON)
-            install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}" 
-                  DESTINATION "${RSTUDIO_INSTALL_RESOURCES}/app"
-                  USE_SOURCE_PERMISSIONS
-                  PATTERN ".gitignore"
-                  EXCLUDE)
-         else()
-            install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}"
-                  DESTINATION "${RSTUDIO_INSTALL_BIN}"
-                  USE_SOURCE_PERMISSIONS
-                  PATTERN "*/share"
-                  EXCLUDE
-                  PATTERN ".gitignore"
-                  EXCLUDE)
-            install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}/share" 
-                  DESTINATION "${RSTUDIO_INSTALL_SUPPORTING}/quarto"
-                  USE_SOURCE_PERMISSIONS
-                  PATTERN ".gitignore"
-                  EXCLUDE)
-         endif()
+         install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}" 
+               DESTINATION "${RSTUDIO_INSTALL_RESOURCES}/app"
+               USE_SOURCE_PERMISSIONS
+               PATTERN ".gitignore"
+               EXCLUDE)
```

Comment:
The conditional on `if (RSTUDIO_ELECTRON)` was dropped in this PR, however `RSTUDIO_INSTALL_RESOURCES` is only set in `cmake/globals.cmake` if `RSTUDIO_ELECTRON` is true.
So: it will try to install into `/app` instead.
I doubt this was intended.

Reply:
The `install` that references `RSTUDIO_INSTALL_RESOURCES` is inside `if(APPLE)` and only Electron is buildable (for installation, at least) on macOS. So, this is working as intended.

```cmake
      if(APPLE)
         install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}" 
               DESTINATION "${RSTUDIO_INSTALL_RESOURCES}/app"
               USE_SOURCE_PERMISSIONS
               PATTERN ".gitignore"
               EXCLUDE)
      else()
         install(DIRECTORY "${RSTUDIO_DEPENDENCIES_QUARTO_DIR}"
               DESTINATION "${RSTUDIO_INSTALL_BIN}"
               USE_SOURCE_PERMISSIONS
               PATTERN ".gitignore"
               EXCLUDE)
      endif()
```

Reply:
I see, thanks for the feedback.